### PR TITLE
update to flat updatepayload structure

### PR DESCRIPTION
**Pull request by juanvallejo**
Updates Dockerfile to include a `/manifests` directory in image.

cc @deads2k @mfojtik @sttts

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,3 +9,6 @@ RUN cd /go/src/github.com/openshift/cluster-openshift-apiserver-operator && go b
 
 FROM centos:7
 COPY --from=0 /go/src/github.com/openshift/cluster-openshift-apiserver-operator/cluster-openshift-apiserver-operator /usr/bin/cluster-openshift-apiserver-operator
+
+COPY manifests/install /manifests
```
Comment (juanvallejo) [inline Dockerfile]: @deads2k would we need to split the single operator.yaml file under the install folder into one file per resource? I'm assuming we'd need to name each file accordingly to ensure some resources are created before others?

Comment (deads2k) [reply Dockerfile]: > @deads2k would we need to split the single operator.yaml file under the install folder into one file per resource? I'm assuming we'd need to name each file accordingly to ensure some resources are created before others?

split whatever is needed.  Make our `manifest` dir be in `manifest`.  Move whatever is needed.

Comment (deads2k) [reply Dockerfile]: Also, ordering is a dangerous precedent, but if it is required, yes.

**Comment by juanvallejo** [comment]
@deads2k thanks, review comments addressed

**Comment by deads2k** [comment]
/lgtm

fix up the controllers too

**Comment by juanvallejo** [comment]
@deads2k 
> fix up the controllers too

Opened pull here: https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/6

**Pull request merged**